.github/workflows/github-actions-tests.yml

```
@@ -7,22 +7,18 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ "3.9", "3.10", "3.11", "3.12" ]
-        poetry-version: [ 1.2 ]
-        pydantic-version: [ 1.10.0, 2.11 ]
         os: [ ubuntu-latest ]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ inputs.python-version }}
```

Should be `matrix` instead, don't know why I put `inputs` in my review on the Beanie repo, probably I saw it in some tutorial so made an unintentional mistake...
```suggestion
      - name: Set up Python ${{ matrix.python-version }}
```